app/views/govuk_publishing_components/components/docs/accordion.yml

```diff
@@ -274,6 +274,35 @@ examples:
             text: "How people read"
           content:
             html: "<p class='govuk-body'>This is the content for How people read.</p>"
+  with_the_anchor_link_navigation:
+    description: |
+      Some apps require custom ids per accordion section heading for linking between those specific sections, sometimes across multiple pages. An example of this is on manuals pages where multiple manuals will each include large sets of accordions and will reference between specific sections for ease of access to that content. [Live example](https://www.gov.uk/guidance/how-to-publish-on-gov-uk/creating-and-updating-pages#associations).
```

Comment:
Further to our chat -  I think accordions on `manuals-frontend` needs more consideration in general. We need to work out how we can do this feature, work out if the API changes can be done.

However for this issue - it's a good fix knowing what we know.